docs/core/entity/number.md

```
@@ -17,7 +17,7 @@ Properties should always only return information from memory and not do I/O (lik
 | mode | string | `auto` | Defines how the number should be displayed in the UI. It's recommended to use the default `auto`. Can be `box` or `slider` to force a display mode.
 | native_max_value | float | 100 | The maximum accepted value in the number's `native_unit_of_measurement` (inclusive)
 | native_min_value | float | 0 | The minimum accepted value in the number's `native_unit_of_measurement` (inclusive)
-| native_step | float | **See below** | Defines the resolution of the values, i.e. the smallest increment or decrement in the number's | native_unit_of_measurement | string | `None` | The unit of measurement that the number's value is expressed in. If the `native_unit_of_measurement` is °C or °F, and its `device_class` is temperature, the number's `unit_of_measurement` will be the preferred temperature unit configured by the user and the number's `state` will be the `native_value` after an optional unit conversion.
+| native_step | float | **See below** | Defines the resolution of the values, that is, the smallest increment or decrement in the number's | native_unit_of_measurement | string | `None` | The unit of measurement that the number's value is expressed in. If the `native_unit_of_measurement` is °C or °F, and its `device_class` is temperature, the number's `unit_of_measurement` will be the preferred temperature unit configured by the user and the number's `state` will be the `native_value` after an optional unit conversion.
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)